pegasus/helpers/form_helpers.rb

```diff
@@ -146,12 +146,11 @@ def insert_form(kind, data, options={})
   form_class = Object.const_get(kind)
   row[:source_id] = form_class.get_source_id(data) if form_class.respond_to? :get_source_id
 
-  if kind == "HocSignup2015"
+  if kind == 'HocSignup2016'
```

Comment:
Of course, we already have (lots of) duplicates.
```
SELECT COUNT(0)
FROM (
  SELECT name, email, COUNT(0) AS cnt
  FROM forms
  WHERE kind = "HocSignup2016"
  GROUP BY 1,2
  HAVING cnt > 1
) t;
+----------+
| COUNT(0) |
+----------+
|     1350 |
+----------+
```
My plan is to wait for @tanyaparker to get back before worrying about data fixes. IIRC, last year we simply ignored all but the last (by creation date).